internal/workerutil/observability.go

```diff
@@ -51,7 +57,7 @@ func NewMetrics(observationContext *observation.Context, prefix string, labels m
 
 	return WorkerMetrics{
 		operations: newOperations(observationContext, prefix, keys, values),
-		numJobs:    numJobs,
+		numJobs:    newLenientConcurrencyGauge(numJobs),
```

Comment:
With the new constructor
```suggestion
		numJobs:    newLenientConcurrencyGauge(numJobs, lenientConcurrencyGaugeInterval),
```

Reply:
Already updated.